modules/osdk-deploy-olm.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+// * operators/operator_sdk/osdk-working-bundle-images.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+endif::[]
+ifeval::["{context}" == "osdk-working-bundle-images"]
+:golang:
+endif::[]
+
+[id="osdk-deploy-olm_{context}"]
+= Deploying an Operator with Operator Lifecycle Manager
+
+Operator Lifecycle Manager (OLM) helps you to install, update, and manage the lifecycle of Operators and their associated services on a Kubernetes cluster. OLM is installed by default on {product-title} and runs as a Kubernetes extension so that you can use the web console and the OpenShift CLI (`oc`) for all Operator lifecycle management functions without any additional tools.
+
+The Operator bundle format is the default packaging method for Operator SDK and OLM. You can use the Operator SDK to quickly run a bundle image on OLM to ensure that it runs properly.
+
+.Prerequisites
+
+- Operator SDK CLI installed on a development workstation
+- Operator bundle image built and pushed to a registry
+- OLM installed on a Kubernetes-based cluster (v1.16.0 or later if you use `apiextensions.k8s.io/v1` CRDs, for example {product-title} {product-version})
+- Logged into the cluster with `oc` using an account with `cluster-admin` permissions
+ifdef::golang[]
+- If your Operator is Go-based, your project must have been updated to use supported images for running on {product-title}
+endif::[]
+
+.Procedure
+
+. Check the status of OLM on your cluster by using the following Operator SDK command:
++
+[source,terminal]
+----
+$ operator-sdk olm status \
+    --olm-namespace=openshift-operator-lifecycle-manager
+----
+
+. Run the Operator on your cluster by using the OLM integration in Operator SDK:
++
+[source,terminal]
+----
+$ operator-sdk run bundle \
+    [-n <namespace>] \//<1>
+    <registry>/<user>/<bundle_image_name>:<tag>
+----
+<1> By default, the command installs the Operator in the currently active project in your `~/.kube/config` file. You can add the `-n` flag to set a different namespace scope for the installation.
++
+This command performs the following actions:
++
+--
+* Create an index image with your bundle image injected. The index image is ephemeral and only intended for development purposes; it is removed during cleanup.
+* Create a catalog source that points to your new index image, which enables OperatorHub to discover your Operator.
+* Deploy your Operator to your cluster by creating an Operator group, subscription, install plan, and all other required objects, including RBAC.
+--
```

Comment:
s/install plan/installation plan (if that is accurate)

Reply:
In this case, it's referring to the `InstallPlan` object, so I'll leave the general reference as-is.

Reply:
Or maybe I'll switch the 3 general references to API object references so I can mark them all up... :thinking:

Reply:
Went with the latter. :v: